### ISSUE TITLE
Develop initial Footer component version

### ISSUE THREAD

**Issue by jeffchew** [carbon-design-system/ibm-dotcom-library#255]
_hahnrob created the following on Jun 20:_  

 As a developer, I need to create the Footer component within the Dotcom components package so that adopters can utilize this component in their application

#### Additional information
 - This issue is dependent on the design and the [package architecture https://github.com/carbon-design-system/ibm-dotcom-library/issues/255](https://zenhub.ibm.com/app/workspaces/digital-design-system-5b3a815a8bf6a932efa6fd03/issues/webstandards/digital-design/860)
 - The priority is documented in the component [Airtable](https://airtable.com/tblYos5iDYwTP6MCK/viw85GYbUVISjrXj9?blocks=hide)
 - Assigned priority:  https://github.com/carbon-design-system/ibm-dotcom-library/issues/255
 - Locale Selector is out of scope for the first release.

#### Acceptance criteria
 - [x] Create an initial component structure for the footer

 

_Original issue: https://github.ibm.com/webstandards/digital-design/issues/1080_

**Comment by jeffchew** [comment]
_On Jul 26, Wonil-Suh1 commented:_
Please use the links from the current site — the same order and label and URL.

**Comment by jeffchew** [comment]
_On Jul 29, james-dow commented:_
Submitted a pull request that basically initializes a basic structure for the Footer, so we can start building it out.

https://github.com/carbon-design-system/ibm-dotcom-library/pull/30

**Comment by jeffchew** [comment]
_On Jul 29, james-dow commented:_
As I worked on the above initial pull request I realized we maybe blocked by #1314. I'm in discussion with @Jeff-Chew and @Kenny-Lam to confirm or not, and figure out how to unblock it if we are.

**Comment by jeffchew** [comment]
_On Aug 01, james-dow commented:_
A number of changes came to the footer yesterday from @Wonil-Suh1.

- Social links are now a list of links within the footer.
- locale selector style is different and is not a part of a bar anymore. (this is out of scope)
- legal links are aligned to the 5th column now
- the lists turn into ~dropdowns~ accordions on small breakpoint

![image](https://media.github.ibm.com/user/19/files/ccf65080-b462-11e9-8b6a-fc9efc3e4163)

![image](https://media.github.ibm.com/user/19/files/b7812680-b462-11e9-855b-33f7425ec503)

**Comment by jeffchew** [comment]
_On Aug 02, Wonil-Suh1 commented:_
Updated footer specs.

### Full version

![dot-com-footer--specs--max](https://media.github.ibm.com/user/25726/files/c973cf80-b51d-11e9-8226-d03ffd0709d5)

![dot-com-footer--specs--xlg](https://media.github.ibm.com/user/25726/files/d09add80-b51d-11e9-86cc-d1a4b2bee1aa)

![dot-com-footer--specs--lg](https://media.github.ibm.com/user/25726/files/d7c1eb80-b51d-11e9-918e-4da4997627f1)

![dot-com-footer--specs--md](https://media.github.ibm.com/user/25726/files/f0320600-b51d-11e9-96e1-8b6b947b4140)

![dot-com-footer--specs--md-max](https://media.github.ibm.com/user/25726/files/eb6d5200-b51d-11e9-98b5-23cf3363f730)

![dot-com-footer--specs--sm](https://media.github.ibm.com/user/25726/files/f627e700-b51d-11e9-92ce-5b573fe28eb5)

![dot-com-footer--specs--sm-max](https://media.github.ibm.com/user/25726/files/f7f1aa80-b51d-11e9-8280-e6eacab8c449)

### Minimal version

![dot-com-footer-minimal--specs--max](https://media.github.ibm.com/user/25726/files/1e174a80-b51e-11e9-8faf-e39b2157a0d5)

![dot-com-footer-minimal--specs--xlg](https://media.github.ibm.com/user/25726/files/21123b00-b51e-11e9-9fca-b8eace6da8d9)

![dot-com-footer-minimal--specs--lg](https://media.github.ibm.com/user/25726/files/86fec280-b51e-11e9-9385-bcd59131ef9d)

![dot-com-footer-minimal--specs--md](https://media.github.ibm.com/user/25726/files/98e06580-b51e-11e9-9a27-47405a8c9805)

![dot-com-footer-minimal--specs--sm](https://media.github.ibm.com/user/25726/files/a0077380-b51e-11e9-857a-6d9e57f3d820)

**Comment by jeffchew** [comment]
_On Aug 02, Wonil-Suh1 commented:_
And the sketch file is [here](https://ibm.box.com/s/w3t9q16xr3y4omuepant96jsjp1rwzml).

**Comment by jeffchew** [comment]
_On Aug 07, Wonil-Suh1 commented:_
@james-dow please use these specs for links:
```
color: $text-02 (gray-30, #bebebe)
hover color: $text-01 (gray-10, #f3f3f3)
active color: $text-04 (white, #ffffff)
```

**Comment by jeffchew** [comment]
_On Aug 07, Wonil-Suh1 commented:_
Updated specs:

### Full version
![dot-com-footer--specs--max](https://media.github.ibm.com/user/25726/files/a322ab00-b8f4-11e9-90cc-9ef259b2d8ab)

![dot-com-footer--specs--xlg](https://media.github.ibm.com/user/25726/files/aae24f80-b8f4-11e9-9e6a-5b165bb48f50)

![dot-com-footer--specs--lg](https://media.github.ibm.com/user/25726/files/b0d83080-b8f4-11e9-8b5e-6ba28bbcb4df)

![dot-com-footer--specs--md-max](https://media.github.ibm.com/user/25726/files/b3d32100-b8f4-11e9-9738-06f71645e29e)

![dot-com-footer--specs--md](https://media.github.ibm.com/user/25726/files/b6ce1180-b8f4-11e9-8564-8ffdbf41611a)

### Short version
![dot-com-footer-minimal--specs--max](https://media.github.ibm.com/user/25726/files/c0577980-b8f4-11e9-905b-fe719ff15c43)

![dot-com-footer-minimal--specs--xlg](https://media.github.ibm.com/user/25726/files/c2b9d380-b8f4-11e9-83a9-d5ad5012e9c7)

![dot-com-footer-minimal--specs--lg](https://media.github.ibm.com/user/25726/files/c51c2d80-b8f4-11e9-84ba-5e4df5d5aebb)

![dot-com-footer-minimal--specs--md](https://media.github.ibm.com/user/25726/files/cbaaa500-b8f4-11e9-9ea3-eec65452d7d6)

![dot-com-footer-minimal--specs--sm](https://media.github.ibm.com/user/25726/files/d402e000-b8f4-11e9-8b34-95b4310b7448)

**Comment by jeffchew** [comment]
_On Aug 08, james-dow commented:_
@Wonil-Suh1: @hahnrob created a new issue for this ticket. #1368

**Comment by jeffchew** [comment]
_On Aug 08, Wonil-Suh1 commented:_
@james-dow interactive state specs:

![dot-com-footer--interactive-specs--links](https://media.github.ibm.com/user/25726/files/02a4b780-b9d6-11e9-8b71-d03530f4c8ed)
![dot-com-footer--interactive-specs--locale-selector](https://media.github.ibm.com/user/25726/files/02a4b780-b9d6-11e9-900c-7f83eccb070a)
![dot-com-footer--interactive-specs--logo](https://media.github.ibm.com/user/25726/files/02a4b780-b9d6-11e9-87b8-a2278f6ba6b8)
